projects/admin/src/app/migration/deduplication/record/brief-view/migration-metadata/migration-metadata.component.ts

```diff
@@ -0,0 +1,168 @@
+  // services
+  protected translateService: TranslateService = inject(TranslateService);
+  // pipes
+  protected mainTitlePipe: MainTitlePipe = inject(MainTitlePipe);
+
+  // current record
+  record = input<any>();
+
+  // current candidate
+  candidate = input<any>();
+
+  // data to load in the table
+  data = computed(() => this.getData());
+
+  /**
+   * Set the color based on the score value.
+   *
+   * @param value score value as string: i.e. (0.68)^2.0 or 1.0.
+   * @returns html code with different text colors.
+   */
+  getScore(value): string {
+    if (value == null) {
+      return ''
+    }
+    if (value?.weight == null) {
+      return `${value.value.toFixed(2)}&nbsp;|&nbsp;1.0`;
+    }
+    var color = 'text-orange-500';
+    if (value.value < 0.6) {
+      color = 'text-red-500';
+    } else if (value.value > 0.8) {
+      color = 'text-green-500';
+    }
+    return `<span class="${color}">${value.value.toFixed(2)}</span>&nbsp;|&nbsp;${value.weight.toFixed(1)}`
+  }
+
+  /**
+   * Get the date for the primeng table.
+   * @returns the list of object for each column and row.
+   */
+  getData(): any[] {
+    // record data
+    const data = this.record()?.metadata?.conversion?.json;
+
+    // candidates data
+    const c_data = this?.candidate()?.json;
+
+    // detailed scores
+    const scores = this?.candidate()?.detailed_score;
+    return [
+      {
+        field: this.record().id ?? '',
+        link: {
+          router: ['/migrations', 'records', 'format', 'migration_data', 'detail', this.record().id],
+          params: { migration: this.record().metadata.migration_id },
+        },
+        c_field: c_data?.pid ?? '',
+        c_link: {
+          router: c_data?.pid ? ['/records', 'documents', 'detail', c_data?.pid] : null,
+        },
+        label: 'pid',
+      },
+      {
+        field: data?.type ? data.type.map((v) => this.translateService.instant(v.main_type)).join('; ') : '',
+        c_field: c_data?.type ? c_data.type.map((v) => this.translateService.instant(v.main_type)).join('; ') : '',
+        label: 'Type',
+        score: this.getScore(scores?.main_type),
+      },
+      {
+        field: data?.title ? this.mainTitlePipe.transform(data.title) : '',
+        c_field: c_data?.title ? this.mainTitlePipe.transform(c_data.title) : '',
+        label: 'Title',
+        score: this.getScore(scores?.title),
+      },
+      {
+        field: data?.sort_date_old ?? '',
+        c_field: c_data?.sort_date_old ?? '',
+        label: 'Publication year',
+        score: this.getScore(scores?.publication_date),
+      },
+      {
+        field: data?.provisionActivity ? data.provisionActivity[0]._text[0].value : '',
+        c_field: c_data?.provisionActivity ? c_data.provisionActivity[0]._text[0].value : '',
+        label: 'Provision activity',
+        score: this.getScore(scores?.provision_activity),
+      },
+      {
+        field: data?.responsibilityStatement
```

Comment:
Display respStatement after title and before sort_date so that it's the same order as a detailed view.